elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -88,7 +92,25 @@ public RequestScope(JsonApiDocument jsonApiDocument,
 
         newPersistentResources = new LinkedHashSet<>();
         commitTriggers = new LinkedHashSet<>();
-        permissionExecutor = new PermissionExecutor(this);
+        if (permissionExecutorClass == null) {
+            this.permissionExecutor = new ActivePermissionExecutor(this);
+        } else {
+            try {
+                PermissionExecutor permissionExecutor;
+                try {
+                    // Try to find a constructor using a request scope first
+                    Constructor<? extends PermissionExecutor> ctor =
```

Comment:
The problem is you cannot instantiate a permission executor without a requestscope (at least, not our executor). We need one of these per request, in general.